core/mention_api.php

```diff
@@ -74,7 +74,7 @@ function mention_get_candidates( $p_text ) {
 			. '(?<!' . $t_quoted_tag . ')' . $t_quoted_tag
 			. ')'
 			# any word char or period, but must not end with period
```

Comment:
Change comment to reflect revised regex

`# any word char, dash or period, must end with word char`